sklearn/utils/multiclass.py

```diff
@@ -234,6 +234,10 @@ def type_of_target(y):
         raise ValueError('Expected array-like (array or non-string sequence), '
                          'got %r' % y)
 
+    sparseseries = (y.__class__.__name__ == 'SparseSeries')
```

Comment:
So, I've gone back to using the name of the class, per prior comments from @amueller on commit [d21c7e38674388f97e146aef67f42bef2fe5d2d2](https://github.com/scikit-learn/scikit-learn/pull/7373/commits/d21c7e38674388f97e146aef67f42bef2fe5d2d2).  Per his comments `pandas` should not be imported at all except in test.